components/MyFooter.js

```diff
@@ -12,7 +12,10 @@ function MyFooter() {
                 width: "100%",
             }}
         >
-            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by <a href="https://lcompilers.org/">LCompilers</a>
+            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by &nbsp;
++            <a href="https://lcompilers.org/">LCompilers</a> | &nbsp;
++            <a href="https://github.com/lfortran/lcompilers_frontend">Contribute</a> |&nbsp;
++            <a href="https://github.com/lfortran/lcompilers_frontend/issues">Report Issues</a>
```

Comment:
```suggestion
            <a href="https://lfortran.org/">LFortran</a> (<a href={"https://github.com/lfortran/lfortran/commit/" + commit.id}>{commit.id}</a>) by &nbsp;
            <a href="https://lcompilers.org/">LCompilers</a> | &nbsp;
            <a href="https://github.com/lfortran/lcompilers_frontend">Contribute</a> |&nbsp;
            <a href="https://github.com/lfortran/lcompilers_frontend/issues">Report Issues</a>
```
Sorry, there is a minor change needed. There are some unneeded `+` symbols in the code here. I noticed them just now.